maqs-appium/src/test/java/com/cognizantsoftvision/maqs/appium/AppiumDriverFactoryUnitTest.java

```diff
@@ -25,7 +24,7 @@
 /**
  * The type Appium driver factory test.
  */
-public class AppiumDriverFactoryUnitTest extends BaseGenericTest {
+public class AppiumDriverFactoryUnitTest extends BaseAppiumTest {
```

Comment:
Why extend base Appium when generic works?

Reply:
reverted back to use baseGenericTest